linux/jdk/redhat/src/main/packaging/temurin/8/temurin-8-jdk.spec

```diff
@@ -1,10 +1,10 @@
-%global upstream_version 8u352-b08
+%global upstream_version 8u362-b09
 # Only [A-Za-z0-9.] allowed in version:
 # https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version
 # also not very intuitive:
 #  $ rpmdev-vercmp 8.0.312.0.1___8 8.0.312.0.0+7
 #  8.0.312.0.0___7 == 8.0.312.0.0+7
-%global spec_version 8.0.352.0.0.8
+%global spec_version 8.0.362.0.0.9
 %global spec_release 2
```

Comment:
this need to be set to 1
if it is the first release on this version

Reply:
lin50 %global upstream_version 8u352-b08-aarch32-20221020 need update for arm32

Reply:
same as for suse

Reply:
all sorted.